zerver/openapi/zulip.yaml

```diff
@@ -7895,6 +7903,13 @@ paths:
           schema:
             type: boolean
           example: true
+        - name: enable_notification_on_unsubscribe_stream
+          in: query
+          description: |
+            Enable private message if someone unsubscribe from private stream
```

Comment:
Enable private message?

Can you describe it in more detail here?

Reply:
How about "Enable notification via private message if someone unsubscribe from private stream"?